apps/web-giddh/src/app/inventory/components/group-stock-report-component/group.stockreport.component.html

```diff
@@ -343,39 +462,47 @@
             <tbody *ngIf="groupStockReport && groupStockReport.count > 0">
                 <tr *ngFor="let txn of groupStockReport.stockReport">
                     <ng-container *ngIf="txn.openingBalance || txn.inwards || txn.outwards || txn.closingBalance">
-                        <td class="bdrB bdrR" colspan="2">
+                        <td class="bdrB bdrR" colspan="2" data-title="Product Name">
                             <span class="justify-content-between d-flex align-items-start">
                                 <span>{{txn.stockName}}</span>
                                 <span class="unit-badge pull-right text-light nowrap">{{txn.openingBalance?.stockUnit}}</span>
                             </span>
                         </td>
                         <!-- in case of all entity with group -->
-                        <td colspan="2" class="bdrB bdrR max-width-140"
+                        <td colspan="2" class="bdrB bdrR max-width-140" data-title="Product Name"
                             *ngIf="GroupStockReportRequest.branchDetails==='allEntity'">
                             {{txn.stockName}}
                         </td>
                         <!-- in case of all entity with group end -->
-                        <td class="bdrB bdrR text-right brNone">
+                        <td data-title="Opening stock -" class="tdmobile-head"></td>
+
+                        <td class="bdrB bdrR text-right brNone" data-title="Qty">
                             <span
                                 [ngClass]="{'text-danger': txn.openingBalance?.quantity<0 }">{{txn.openingBalance?.quantity}}</span>
                         </td>
-                        <td class="bdrB bdrR text-right">{{txn.openingBalance?.amount}}</td>
+                        <td class="bdrB bdrR text-right" data-title="Value">{{txn.openingBalance?.amount}}</td>
 
-                        <td class="bdrB bdrR text-right brNone">
+                        <td data-title="inwards -" class="tdmobile-head"></td>
```

Comment:
data-title="inwards **-**  can we remove **-** hyphen

Reply:
@shalinee-walkover It is a heading which has sub text, it should be visible properly and different that's why I used hyphen.